### PR TITLE
Add SMP missing release notes

### PR DESCRIPTION
**Pull request by krishi0408**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Comment by krishi0408** [comment]
@SushrutHarness please review it once.

File: release-notes/continuous-delivery.md [hunk 1]
```diff
@@ -307,7 +309,34 @@ on class `ScriptSshExecutor.java` made the log stream terminate.
...
   - This combination might cause issues if two steps with same account, org, project, and provisioner identifier are running simultaneously on same delegate.
   - With this change every Terraform step execution will run in a unique working directory.
-- Console logs missing from CloudFormation steps. (CDS-84962, ZD-53810, ZD-53865) There was an issue where CloudFormation steps were not updating the console longs when there are multiple steps with a similar prefix. This issue is now fixed.
+- Console logs missing from CloudFormation steps. There was an issue where CloudFormation steps were not updating the console longs when there are multiple steps with a similar prefix. This issue is now fixed.(CDS-84962, ZD-53810, ZD-53865)
+- When internal looping strategy fields marked as runtime, the run pipeline form did not prompt for input. Condition updated to validate looping strategy runtime inputs. (CDS-86445)
+- Blank tile on ASG traffic shifter. Blank tile on the UI for the ASG Traffic Shifter fixed. (CDS-86298)
+- Customer unable to set environment infrastructure definition. Infrastructure definition now set with provisioner step group template. (CDS-86109, ZD-54775)
```
Comment (brian-f-harness) [inline release-notes/continuous-delivery.md:315]:
```suggestion
- Infrastructure definitions didn't allow you to set an environment. (CDS-86109, ZD-54775)

   Infrastructure definitions now allow you to set the provisioner step group template.
```

File: release-notes/continuous-delivery.md [hunk 2]
```diff
@@ -307,7 +309,34 @@ on class `ScriptSshExecutor.java` made the log stream terminate.
...
+- Unsaved Changes Doesn't Display Whitespace Changes. Whitespace difference consideration implemented in diff editors. (CDS-85964)
```
Comment (brian-f-harness) [inline release-notes/continuous-delivery.md:316]:
```suggestion
- Diff editors didn't display unsaved whitespace changes. (CDS-85964)

   Whitespace difference consideration has been implemented in diff editors to resolve this issue.
```

**Review by brian-f-harness — COMMENTED** [review]
@krishi0408 I added some suggestions up to line 318. Please revise the remaining RN, using the same format (past tense, full sentences where possible, code format where necessary), and then I'll take another look.

This page is a good resource for more info: https://harness.atlassian.net/wiki/spaces/DOCS/pages/1689845830/Release+notes+process+and+guidelines#What-to-include-in-release-notes

**Comment by brian-f-harness** [comment]
@krishi0408 Could you please revisit this PR as time allows? It will help with release note accuracy.

**Comment by brian-f-harness** [comment]
@krishi0408 These items need to be documented for the applicable RN. Did you open a new PR? If not, please open a new PR to add the items.

**Pull request closed without merging**